mantidproject/mantid
# Updates to release notes v4.2 Step 4

**Description of work.**

Reformatting some of Index page.
Fixing a typo in Mantid WB page.
Diffraction page - correcting links.
Resizing Images.
Add ALFview image to Direct_geometry page
Replace Data Analysis interface on Indirect_geometry

**To test:**

Build docs.html
On Diffraction page, Check links to PredictFractionalPeaks and IndexPeaks in the Single Crystal>Improvements section 

refs #27100 

*This does not require release notes* because **It is release notes!!**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
